ggml.c

```diff
@@ -15858,7 +15858,7 @@ struct ggml_cgraph * ggml_build_forward_ctx(struct ggml_context * ctx, struct gg
 }
 
 size_t ggml_graph_overhead(void) {
-    return GGML_OBJECT_SIZE + GGML_GRAPH_SIZE + 16;
+    return GGML_OBJECT_SIZE + ((GGML_GRAPH_SIZE + GGML_MEM_ALIGN - 1)/GGML_MEM_ALIGN)*GGML_MEM_ALIGN;
```

Comment:
Btw, we should add a `ggml_pad` helper function since this operation is common.
It could even be in the header as macro `GGML_PAD` so we can reuse it in the backends